sklearn/model_selection/_search.py

```diff
@@ -618,15 +618,16 @@ def fit(self, X, y=None, groups=None, **fit_params):
             if self.refit is not False and (
                     not isinstance(self.refit, six.string_types) or
                     # This will work for both dict / list (tuple)
-                    self.refit not in scorers):
+                    self.refit not in scorers) and not callable(self.refit):
```

Comment:
shouldn't the ')' after `scorers` be at the end of the line?

Reply:
I think the ')' should be where it is right now. The issue might be caused by purest.raises. As we can see commit 561ab54 successfully passed all tests using 'assert_raise_message' func instead of purest.raises(...)

Reply:
Nvm, now it's working ;) @adrinjalali